src/eosjs-api.ts

```diff
@@ -6,75 +6,15 @@
 
 "use strict";
 
-import { Abi, GetInfoResult, JsonRpc, PushTransactionArgs } from "./eosjs-jsonrpc";
+import { JsonRpc } from "./eosjs-jsonrpc";
 import * as ser from "./eosjs-serialize";
+import { AuthorityProvider, AbiProvider, BinaryAbi, CachedAbi, SignatureProvider } from "./eosjs-api-interfaces";
+import { Abi, GetInfoResult, PushTransactionArgs } from "./eosjs-rpc-interfaces";
 
 // tslint:disable-next-line
-const abiAbi = require('../src/abi.abi.json');
+const abiAbi = require('./abi.abi.json');
```

Comment:
I wish there was a better name for this, but I understand that the structure is abi.abi, correct?

Reply:
An ABI describes how to convert from binary to json and back. The ABI ABI describes how to convert an ABI back and forth.